src/STAT_FrontEnd.C

```diff
@@ -880,44 +575,12 @@ StatError_t STAT_FrontEnd::launchMrnetTree(StatTopology_t topologyType, char *to
             daemonArgv[daemonArgc - 1] = NULL;
         }
 #endif
-
-        printMsg(STAT_LOG_MESSAGE, __FILE__, __LINE__, "\tSetting up daemon args for serial attach and MRNet launch\n");
```

Comment:
Moved verbatim to STAT_lmonFrontEnd::addDeamonSerialProcArgs.   The cti version doesn't handle the serial attach.